index.js

```diff
@@ -0,0 +1,36 @@
+const express = require("express");
+require("body-parser");
+
+const errorHandler = require("./middlewares/error-handler.middleware");
+const routeNotFound = require("./middlewares/route-not-found-handler.middleware");
+
+const user = require("./routers/user.route");
+const quiz = require("./routers/quiz.route");
+const leaderBoard = require("./routers/leaderBoard.route");
+
+const app = express();
+app.use(express.json());
+
+const cors = require("cors");
+app.use(cors());
+
+const dbConnect = require("./db/db.connect");
```

Comment:
Make all require statements at the top. Instead of after initiating server :)

Reply:
done